README.md

```diff
@@ -29,15 +29,20 @@ Others:
 - [skype](https://github.com/snaggen/skype-app) -  2017
 
 ## Tools
-
+### App Stores
+- [Souk](https://gitlab.gnome.org/haecker-felix/souk) - Souk is a flatpak-based App Store, written with GTK4 and Rust.
+- [bauh](https://github.com/vinifmor/bauh) - Graphical user interface for installing Linux applications. Supports AppImage, Arch packages (AUR, Pacman), Debian packages, Flatpak, Snap and native Web applications
+- [Zorin Store](https://zorin.com/os/) - Zorin OS Store comes pre-loaded with the app catalogs of Flathub, the Snap store, and the Ubuntu & Zorin OS APT repositories, AppImage packages and Windows apps (optionally using Windows App Support, powered by WINE)
+### Another Tools
 - [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal) -  Permissions manager for Flatpak Apps
-- rpm2flatpak
-- [game-to-flatpak](https://github.com/hadess/flatpak-games) - 2018
```

Comment:
Why remove rpm2flatpak and game-to-flatpak?

Reply:
cause this looks like obandoned project